### PR TITLE
Release/bee hummingbird

### PR DESCRIPTION
**Pull request by fblundun**

### CODE REVIEW

**Comment by fblundun** [comment]
I haven't yet squashed all the Kinesis LZO S3 Sink commits together - I will do once we have decided we are happy with them.

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/loaders/ThriftLoader.scala [hunk 1]
```diff
@@ -58,53 +60,144 @@ object ThriftLoader extends Loader[Array[Byte]] {
...
-            headers,
-            networkUserId,
-            CollectorApi.SnowplowTp1, // No way of storing API vendor/version in Thrift yet, assume Snowplow TP1
-            None, // No way of storing content type in Thrift yet
-            None  // No way of storing request body in Thrift yet
-          )
-        )
+      if (schema.isSetSchema) {
+        schema.getSchema match {
+          case ExpectedSchema => convertSchema1(line)
```
Comment (alexanderdean) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/loaders/ThriftLoader.scala]: Should we do this with a `SchemaCriterion`?

File: 3-enrich/scala-hadoop-enrich/project/Dependencies.scala [hunk 2]
```diff
@@ -13,12 +13,15 @@
...
-  
+
   val resolutionRepos = Seq(
     // Required for our json4s snapshot
     "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/",
-     // For some misc Scalding and Twitter libs
+    // For some misc Scalding and Twitter libs
     "Concurrent Maven Repo" at "http://conjars.org/repo",
+    "Clojars Maven Repo" at "http://clojars.org/repo",
+    // For twitter libs
```
Comment (alexanderdean) [inline 3-enrich/scala-hadoop-enrich/project/Dependencies.scala]: Twitter not twitter

File: 3-enrich/scala-hadoop-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich.hadoop/EtlJob.scala [hunk 3]
```diff
@@ -186,4 +192,19 @@ class EtlJob(args: Args) extends Job(args) {
...
+   *
+   * @param format the inFormat configuration
+   * @param path
+   * @return FixedPathLzoRaw for LZO-compressed thrift, otherwise MultipleTextFiles
+   */
+  def getInputPipe(format: String, path: String): Pipe = {
+    import TDsl._
+    format match {
+      case "thrift" => FixedPathLzoRaw(path).toPipe('line)
+      case _ => MultipleTextLineFiles(path).read
```
Comment (alexanderdean) [inline 3-enrich/scala-hadoop-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich.hadoop/EtlJob.scala]: Nice

File: 4-storage/kinesis-lzo-s3-sink/README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
+# Kinesis LZO S3 Sink
+
+## Introduction
+
+The Kinesis LZO S3 Sink consumes records from an [Amazon Kinesis][kinesis] stream, compresses them using [splittable LZO][hadoop-lzo], and writes them to S3.
```
Comment (alexanderdean) [inline 4-storage/kinesis-lzo-s3-sink/README.md]: Can we add a sentence about how we handle _framing_ (where one message ends and the next begins)? Explain how we use the Elephant Bird project's `BinaryBlockWriter`, and the Protocol Buffer array. Will save somebody a lot of pain...

File: 4-storage/kinesis-lzo-s3-sink/project/BuildSettings.scala [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+import sbt._
+import Keys._
+
+object BuildSettings {
+
+  // Basic settings for our app
+  lazy val basicSettings = Seq[Setting[_]](
+    organization          :=  "com.snowplowanalytics",
+    version               :=  "0.1.0",
+    description           :=  "Kinesis sink for S3",
```
Comment (alexanderdean) [inline 4-storage/kinesis-lzo-s3-sink/project/BuildSettings.scala]: Make description more descriptive - LZO etc

File: 4-storage/kinesis-lzo-s3-sink/project/Dependencies.scala [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    val commonsLang3         = "3.1"
+  }
+
+  object Libraries {
+    // Java
+    val slf4j                = "org.slf4j"                  %  "slf4j-simple"             % V.slf4j
+    val kinesisClient        = "com.amazonaws"              %  "amazon-kinesis-client"    % V.kinesisClient
+    val kinesisConnector     = "com.amazonaws"              %  "amazon-kinesis-connector" % V.kinesisConnector
+    val hadoop               = "org.apache.hadoop"          %  "hadoop-core"              % V.hadoop
+    val elephantbird         = "com.twitter.elephantbird"   % "elephant-bird-core"        % V.elephantbird
```
Comment (alexanderdean) [inline 4-storage/kinesis-lzo-s3-sink/project/Dependencies.scala]: %  "elephant-bird"...

(whitespace)

File: 4-storage/kinesis-lzo-s3-sink/project/Dependencies.scala [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    // Thrift (test only)
+    val collectorPayload     = "com.snowplowanalytics"      % "collector-payload-1"       % V.collectorPayload % "test"
```
Comment (alexanderdean) [inline 4-storage/kinesis-lzo-s3-sink/project/Dependencies.scala]: %  "collector-payload-1" 

(whitespace)

File: 4-storage/kinesis-lzo-s3-sink/src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SnowplowRawEventTransformer.scala [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+import org.apache.commons.codec.binary.Base64
+
+// Scalaz
+import scalaz._
+import Scalaz._
+
+/**
+ * Thrift serializer/deserializer class
+ */
+class SnowplowRawEventTransformer extends ITransformer[ ValidatedRecord, EmitterInput ] {
```
Comment (alexanderdean) [inline 4-storage/kinesis-lzo-s3-sink/src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SnowplowRawEventTransformer.scala]: Should this start with `Snowplow`? It's not really Snowplow-opinionated...

File: CHANGELOG [hunk 9]
```diff
@@ -1,3 +1,39 @@
+Version 60 (2015-01-xx)
```
Comment (alexanderdean) [inline CHANGELOG]: Release 60 Bee Hummingbird (2015-01-xx)

**Pull request closed without merging**